CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## Features
 
+- replaced rounded faraday constant with it's exact value in bpx.py for better comparison between different tools
```

Comment:
```suggestion
- Replaced rounded faraday constant with its exact value in `bpx.py` for better comparison between different tools
```

Reply:
Additionally, could you please move it to the unreleased section, @dion-w?

Reply:
yeah sure